src/streaming-services/nrk/NrkApi.ts

```diff
@@ -100,51 +170,63 @@ class _NrkApi extends Api {
 		}
 	};
 
-	parseHistoryItem = (historyItem: NrkHistoryItem): Item => {
-		const program: NrkProgramInfo = historyItem.program;
-		let item: Item;
+	parseHistoryItem = async (historyItem: NrkProgressItem): Promise<Item> => {
 		const serviceId = this.id;
-		const id = program.id;
-		const type = program.programType === 'Episode' ? 'show' : 'movie';
-		const year = program.productionYear;
-		const percentageWatched = parseInt(historyItem.lastSeen.percentageWatched, 10);
-		const watchedDate = this.convertAspNetJSONDateToDateObject(historyItem.lastSeen.at);
-		const watchedAt = watchedDate ? moment(watchedDate) : undefined;
+		const id = historyItem.id;
+		const programInfo = historyItem._embedded.programs;
+		const programPage = await this.lookupNrkItem(programInfo._links.self.href);
+		const type = programPage._links.seriesPage !== undefined ? 'show' : 'movie';
+		const titleInfo = programInfo.titles;
+		const title = programPage.moreInformation.originalTitle ?? titleInfo.title;
```

Comment:
When testing I've figured out that sometimes the title would have been found if the title from titleInfo was used instead. A bit annoying since it seems safer to go with originalTitle, since translations is often missing on trakt.
originalTitle sometimes includes seasons, and even episode title, but it's rather rare, so not sure if it's worth making special code for it.

Reply:
Perhaps we should use a fallback system in this situation? Try first with `originalTitle` and if no results are found try with the other one?

To make this more generic, we could add a new field to `Item` called `alternativeTitles` or something, and this would be an array that is tried, in order, if the main title didn't match.

Reply:
And then all we would need to do is modify `TraktSearch.find` to do a loop that only returns when an item is found or it goes through all titles.

Reply:
This seems like a good idea, but not really a job for this PR.

Reply:
Yeah, I know, I was just throwing it out there.